src/NHN.DtoContracts/NHN.DtoContracts/Flr/Data/PrimaryHealthCarePerson.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Runtime.Serialization;
+using NHN.DtoContracts.Common.en;
+
+namespace NHN.DtoContracts.Flr.Data
+{
+    /// <summary>
+    /// Basisdata for en person.
+    /// </summary>
+    [DataContract(Namespace = Namespaces.FlrV1)]
+    [Serializable]
+
+    public class PrimaryHealthCarePerson
+    {
+        /// <summary>
+        /// Identifikator for primærhelseteamperson.
```

Comment:
Identifikator for en person som er medlem av et primærhelseteam (vet ikke om det ble noe bedre.. ?)